### ISSUE TITLE
Z-hop not working in v2 beta

### ISSUE THREAD

**Issue by smwoodward** [SoftFever/OrcaSlicer#4494]
### Is there an existing issue for this problem?

- [X] I have searched the existing issues

### OrcaSlicer Version

2.0.0

### Operating System (OS)

macOS

### OS Version

14.3.1

### Printer

Ratrig 

### How to reproduce

Look for the G1 Z**** in the gcode, I was not able to find any z movement codes with z hop on both with and without firmware retraction.

### Actual results

Z hop isn't in gcode

### Expected results

z hop should be in the zip code

### Project file & Debug log uploads

[debug.3mf.zip](https://github.com/SoftFever/OrcaSlicer/files/14608694/debug.3mf.zip)
[log.zip](https://github.com/SoftFever/OrcaSlicer/files/14608695/log.zip)

### Checklist of files to include

- [X] Log file
- [X] Project file

**Comment by Naahuel** [comment]
I came here looking if someone had reported this because it was driving me crazy. I couldn't make z hop work! No matter how much I tweaked the settings, nothing. 

I was collecting the evidence to add to your bug report when I realized what happened. I had changed the setting overrides for my material profile. 

I opened your debug.3mf project and what do you know...
![image](https://github.com/SoftFever/OrcaSlicer/assets/1612488/cb3fd9a7-ae57-4ad4-a7cd-eccc7566a633)

Thank you for making me find my problem and I hope this fixed yours as well!

**Comment by smwoodward** [comment]
> I came here looking if someone had reported this because it was driving me crazy. I couldn't make z hop work! No matter how much I tweaked the settings, nothing.
> 
> I was collecting the evidence to add to your bug report when I realized what happened. I had changed the setting overrides for my material profile.
> 
> I opened your debug.3mf project and what do you know... ![image](https://private-user-images.githubusercontent.com/1612488/313033603-cb3fd9a7-ae57-4ad4-a7cd-eccc7566a633.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTA0NjQ1NjUsIm5iZiI6MTcxMDQ2NDI2NSwicGF0aCI6Ii8xNjEyNDg4LzMxMzAzMzYwMy1jYjNmZDlhNy1hZTU3LTRhZDQtYTdjZC1lY2NjNzU2NmE2MzMucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMxNSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMTVUMDA1NzQ1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZjlkNjUwNWJjNDQ0YzE3MWExNTk0MWFlYzYzMDZmZDMyZTVjMzI4YjExNjZjZThjZWFkZTI4YjMwYjRiYWY4OSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.yfG1v93oM1sQH7YSlSd8NC3eSNA0ToaelQYSA27yb0Y)
> 
> Thank you for making me find my problem and I hope this fixed yours as well!

🤦🏼‍♂️🤦🏼‍♂️🤦🏼‍♂️🤦🏼‍♂️🤦🏼‍♂️🤦🏼‍♂️🤦🏼‍♂️. Really......  That's what the issue was?  Now I feel dumb.

Yeah I just verified that setting was set, and also verified in the gcode by removing it, now I see the z hop.  

Well maybe we can turn this from being a bug in OrcaSlicer to being a bug in the included profiles.  lol

**Comment by Naahuel** [comment]
Happens to the best of us, it was a huge coincidence you reported it first because I came here to do that. It was driving me nuts.


> Well maybe we can turn this from being a bug in OrcaSlicer to being a bug in the included profiles. lol


Yeah well I changed mine too because it came with 0.4mm as the default and I remember I couldn't see why it was z hopping! Then I forgot I fixed that on the material profile and now that I WANTED to use z hop I couldn't remember this.
